src/templates/Resources/PractitionerRole.hbs

```diff
@@ -4,31 +4,40 @@
         "resourceType": "PractitionerRole",
         "id":"{{ID}}",
         {{>DataType/XCNPractitionerRole.hbs XCN=PD1-4}},
-        {{>DataType/XCNPractitionerRole.hbs XCN=PRT-5}},
-        "specialty":
-        {
-            {{>DataType/CWECodeableConcept.hbs CWE=PRT-6}},
-        },
+        "identifier":
+        [
+            { {{>DataType/EIIdentifier.hbs EI=ROL-1}} },
+            {
+                {{>DataType/PLN.hbs PLN=PRT-24}},
+            },
+        ],
+        "code":
+        [
+            { {{>DataType/CWECodeableConcept.hbs mapping="CodeSystem/Role.hbs" CWE=ROL-3}} },
+        ],
+        {{>DataType/XCNPractitionerRole.hbs XCN=ROL-4}},
         "period":
         {
-            "start":"{{convertDateTimeStringToUTC PRT-11}}",
-            "end":"{{convertDateTimeStringToUTC PRT-12}}",
+            "start":"{{formatAsDateTime ROL-5}}",
+            "start":"{{formatAsDateTime PRT-11}}",
+            "end":"{{formatAsDateTime ROL-6}}",
+            "end":"{{formatAsDateTime PRT-12}}",
         },
         "telecom":
         [
+            { {{>DataType/XTN.hbs XTN=ROL-12}} },
             { {{>DataType/XTN.hbs XTN=PRT-15}} },
         ],
-        "identifier":
+        {{>DataType/XCNPractitionerRole.hbs XCN=OBX-25}},
+        {{>DataType/XCNPractitionerRole.hbs XCN=PRT-5}},
+        "specialty":
         [
-            {
-                {{>DataType/PLN.hbs PLN=PRT-24}},
-            },
+            { {{>DataType/CWECodeableConcept.hbs CWE=PRT-6}} },
         ],
-        {{>DataType/XCNPractitionerRole.hbs XCN=OBX-25}},
+        {{>DataType/XCNPractitionerRole.hbs XCN=ORC-12}},

```

Comment:
I dont think ORC12 has practitioner role, just practitioner

Reply:
Probably its coming from this spreadsheet : https://docs.google.com/spreadsheets/d/11VBF5TIRJTfRwkRIK1XH-k40dZqCd3g3a0sBY-zJU4Q/edit#gid=0

Reply:
Looks good!